release/nightly_tests/dataset/streaming_split_benchmark.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+from typing import Optional
+
+from benchmark import Benchmark
+import ray
+
+
```

Comment:
Block comment on the purpose of the test will be good.

Reply:
Addressed